front/components/sparkle/AppLayout.tsx

```diff
@@ -124,7 +126,10 @@ export default function AppLayout({
         <div className="relative h-full w-full flex-1 flex-col overflow-x-hidden overflow-y-hidden">
           <main
             id={CONVERSATION_PARENT_SCROLL_DIV_ID.page}
-            className="flex h-full w-full flex-col items-center"
+            className={classNames(
+              "flex h-full w-full flex-col items-center overflow-y-auto",
+              titleChildren ? "" : hasTopPadding ? "lg:pt-8" : ""
```

Comment:
I would rather keep hasTopPadding optional and infer from titleChildren if it's not set, to make sure that `hasTopPadding` is taken into account if passed :
```suggestion
              (hasTopPadding ?? !titleChildren) ? "lg:pt-8" : ""
```